specification/http/src/main/scripts/org/kaazing/specification/http/rfc7230/connection.management/client.with.pipelining.must.not.retry.pipelining.immediately.after.failure/request.rpt

```diff
@@ -29,6 +29,7 @@ read "HTTP/1.1 200 OK\r\n"
 read "Content-Length: 8\r\n"
 read "\r\n"
 read "request1"
+closed
```

Comment:
Why not use a `read closed` to keep it in the http layer.  Closed will wait for tcp layer to close, I think?? also, the ordering looks weird to me, could we put the read notify above the read close or, tie it to the closed (`closed notify` this might not be implemented yet?)?